src/develop/security/end-user-manage/end-user-authentication/configure-azuread.md

```diff
@@ -190,6 +190,29 @@ Do the following:
 
 You now have permissions to grant OutSystems roles to users. Check [Grant a role to an end user](../end-user-roles.md#grant-role) for detailed instructions.
 
+### Configure group roles 
```

Comment:
@nunojrsilva, can you have a look?

Reply:
All good, we can proceed with the merge 👍

Reply:
Thank you for reviewing the pull request. Because of the restructuring of the documentation, I'll be including this new material in a different pull request.